jbpm-test-coverage/src/main/java/org/jbpm/test/jobexec/CheckCallCommand.java

```diff
@@ -0,0 +1,40 @@
+package org.jbpm.test.jobexec;
+
+import org.kie.api.executor.Command;
+import org.kie.api.executor.CommandContext;
+import org.kie.api.executor.ExecutionResults;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CheckCallCommand implements Command {
+    
+    private static final Logger logger = LoggerFactory.getLogger(CheckCallCommand.class);
+
+    private static Boolean commandExecuted = false;
+    private static Object LOCK = new Object();
```

Comment:
A CyclicBarrier or CountDownLatch would (I think?) eliminate the need to have both a boolean and a LOCK, and also eliminate the need for a sleep or wait on the LOCK field. 

If you want, I can go ahead and merge it and then change the code to show you how I would do it?